website/pages/docs/deployment/kestra.mdx

```diff
@@ -1,102 +1,86 @@
 
-## Step 2: Set up a PostgreSQL database
+```bash
+docker run --pull=always --rm -it -p 8080:8080 --user=root -v /var/run/docker.sock:/var/run/docker.sock -v /tmp:/tmp kestra/kestra:latest-full server local
+```
 
-We will use a PostgreSQL database as a destination for our CloudQuery syncs. You can use any PostgreSQL database, but for this tutorial we will use a Docker container.
+Once the container is running, open http://localhost:8080 in your browser.  
 
-```bash copy
-docker run --name cloudquery-postgres -e POSTGRES_PASSWORD=pass -p 5432:5432 -d postgres
-``` 
 
 ## Step 2: Create a Kestra flow
 
-Inside the Kestra UI, go to the `Flows` tab and click on `Create`. You can now create a new flow with the following content:
+Inside the Kestra UI, go to the `Flows` page and click on `Create`. You can now paste the following content:
 
 ```yaml copy
-id: "cloudquery"
-namespace: "io.kestra"
+id: cloudquery_sync
+namespace: dev
+
 tasks:
-- id: "bash"
-  type: "io.kestra.core.tasks.scripts.Bash"
-  runner: DOCKER
-  inputFiles:
-    config.yml: |
-      kind: source
-      spec:
-        name: aws
-        path: cloudquery/aws
-        registry: cloudquery
-        version: "VERSION_SOURCE_AWS"
-        tables: ["aws_ec2*"]
-        destinations: ["postgresql"]
+  - id: hn_to_csv
+    type: io.kestra.plugin.cloudquery.Sync
+    incremental: false
+    outputFiles:
+      - "cloudquery/**"
+    configs:
+      - kind: source
         spec:
-      ---
-      kind: destination
-      spec:
-        name: "postgresql"
-        path: "cloudquery/postgresql"
-        registry: cloudquery
-        version: "VERSION_DESTINATION_POSTGRESQL"
-        write_mode: "overwrite-delete-stale"
+          name: hackernews
+          path: cloudquery/hackernews
+          registry: cloudquery
+          version: v3.0.21
+          tables: ["*"]
+          destinations:
+            - file
+          spec:
+            item_concurrency: 100
+            start_time: "2024-01-22T00:00:00Z"
+      - kind: destination
         spec:
-          connection_string: ${PG_CONNECTION_STRING}
-  dockerOptions:
-    image: ghcr.io/cloudquery/cloudquery:latest
-    entryPoint: [""]
-  warningOnStdErr: false
-  env:
-    PG_CONNECTION_STRING: "postgresql://postgres:pass@host.docker.internal:5432/cloudquery?sslmode=disable"
-  commands:
-  - '/app/cloudquery sync {{ workingDir }}/config.yml --log-console'
+          name: file
+          path: cloudquery/file
+          version: v3.4.17
+          spec:
+            path: "cloudquery/{{TABLE}}/{{UUID}}.{{FORMAT}}"
+            format: csv
 ```
 
-We are using the Docker runner with a `Bash` task to run the `cloudquery sync` command. The `inputFiles` section allows us to pass a configuration file to the task. It is also possible to read this configuration file from disk or a remote location, but we will keep it simple for now.
-
-import { Callout } from 'nextra-theme-docs';
```

Comment:
I believe you need to keep this import to use the `Callout` component. That should be the failing Vercel build

Reply:
thanks, done